src/siema.js

```diff
@@ -45,11 +45,8 @@ export default class Siema {
       onInit: () => {},
       onChange: () => {},
     };
-    const userSttings = options;
-    for (const attrname in userSttings) {
-      settings[attrname] = userSttings[attrname];
-    }
-    return settings;
+
+    return Object.assign(settings, options);
```

Comment:
It's better to use `Object.assign` but you'll have to deal with lack of browser support https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign.
If is not possible I can revert this change.